charts/dial/examples/generic/simple/README.md

```diff
@@ -3,9 +3,10 @@
 - [AI DIAL Generic Installation Simple Guide](#ai-dial-generic-installation-simple-guide)
   - [Prerequisites](#prerequisites)
   - [Expected Outcome](#expected-outcome)
+  - [What's next?](#whats-next)
```

Comment:
This section is for additional application configuration. It seems odd to place it above the installation section.